source/sass/style.scss

```
@@ -1 +1,42 @@
-// Начните писать стили здесь
+@import "fonts.scss";
+@import "common.scss";
+@import "./page.scss";
```

А почему у тебя пути написаны в разном стиле?
Почти везде с точкой путь. Надо тогда не почти везде, а везде сделать =)
```suggestion
@import "./fonts.scss";
@import "./common.scss";
@import "./page.scss";
```

Ну или поудалять везде. Тут как хочешь